pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go

```diff
@@ -103,9 +103,10 @@ func (c *UpgradeBackupController) sync(ctx context.Context, syncCtx factory.Sync
 	clusterOperatorObj := originalClusterOperatorObj.DeepCopy()
 	if !isBackupConditionExist(originalClusterOperatorObj.Status.Conditions) {
 		configv1helpers.SetStatusCondition(&clusterOperatorObj.Status.Conditions, configv1.ClusterOperatorStatusCondition{
-			Type:   "RecentBackup",
-			Status: configv1.ConditionUnknown,
-			Reason: "ControllerStarted",
+			Type:    "RecentBackup",
+			Status:  configv1.ConditionUnknown,
+			Reason:  "ControllerStarted",
+			Message: "Due to a minor version bump, etcd will be taking a snapshot shortly",
```

Comment:
that's quite nit-picky and I'm not super familiar with this controller yet, but how do you know it's a minor version bump?